packages/console/src/pages/Connectors/types.ts

```diff
@@ -5,7 +5,7 @@ export type ConnectorFormType = {
   logoDark: string;
   target: string;
   syncProfile: SyncProfileMode;
-};
+} & Record<string, number | boolean | string>; // Extend custom connector config form
```

Comment:
should this be a separate form type?

Reply:
Replaced with "unknown" to avoid deep type checks.